src/html.js

```diff
@@ -0,0 +1,38 @@
+// File contents copied from Gatsby's cache in order to add a link tag for the font
+// Docs: https://www.gatsbyjs.org/docs/custom-html/
```

Comment:
I noticed that the layout component was re-rendering a bit. That shouldn't happen, but I couldn't figure out why -- the only real issue with it was that the Google font stylesheet kept getting reimported because the styled-components `GlobalStyle` was re-rendering. This caused the fonts to flicker on every page change which is annoying.

This file is a lil workaround for that -- it allows us to import the fonts using a standard `link` tag. Now the flicker is gone 👌 

I documented the reasoning for the file at the top as well as provided a link to the docs on Gatsby's site.